DirectProgramming/DPC++/Jupyter/sycl-performance-portability-training/00_Introduction.ipynb

```diff
@@ -388,7 +388,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.10"
+   "version": "3.9.7"
+  },
+  "vscode": {
```

Comment:
@KanclerzPiotr Can you remove vscode?

Reply:
Removed the metadata changes with vscode